workers/ohsome_quality_analyst/indicators/building_completeness/indicator.py

```diff
@@ -271,3 +270,17 @@ async def get_hex_cells(feature: Feature) -> FeatureCollection:
     if feature_collection["features"] is None:
         raise HexCellsNotFoundError
     return feature_collection
+
+
+async def get_shdi(featurecollection: FeatureCollection) -> list:
```

Comment:
it's probably very unlikely, but what if we just get single hex cell without SHDI value? then you would not be able to calculate the mean.

Reply:
We agreed that we get an Exception at the moment. In the future we might handle the issue differently.